Sources/PropertyEditor.swift

```diff
@@ -0,0 +1,680 @@
+import Result
+
+/// A mutable, observable property that has an optionally failable action
+/// associated with the setter.
+///
+/// ## Consistency when nested
```

Comment:
I'm not sure I understand this section fully. Are you suggesting that a change in the underlying property might cause a failure to then get reported? The test cases appear to suggest this is the case, but the comment here is not so helpful.

Perhaps an inline, simple example would be helpful, similar to what we see all over the standard library comments.